aten/src/ATen/native/native_functions.yaml

```diff
@@ -1936,14 +1936,17 @@
     SparseCPU: new_with_tensor_and_size_sparse
     SparseCUDA: new_with_tensor_and_size_sparse
 
-- func: sparse_coo_tensor(Type dtype, IntList size) -> Tensor
-  variants: []
-
 - func: sparse_coo_tensor(IndexTensor indices, Tensor values) -> Tensor
-  variants: []
 
 - func: sparse_coo_tensor(IndexTensor indices, Tensor values, IntList size) -> Tensor
-  variants: []
+
+# FIXME: would be nicer if TensorOptions was optional based; not adding default arguments for options given
+# the default would never make sense.
+- func: sparse_coo_tensor(IntList size, *, TensorOptions options) -> Tensor
+
+- func: sparse_coo_tensor(IndexTensor indices, Tensor values, *, TensorOptions options) -> Tensor
```

Comment:
This entry and the one below don't make much sense to me. The type of the sparse tensor in these cases are already defined by the `indices` and `values` types. The type isn't actually used!! See
https://github.com/pytorch/pytorch/blob/16e21e14e398b90cfada8a0c6a2f628505e1dfac/aten/src/ATen/native/sparse/SparseTensor.cpp#L151

Can I purge it? Context: I'm rebasing #11253